src/theme/icons/components/ShoppingBagIcon.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { Path } from "react-native-svg";
+import { COLOR_TEXT_WHITE } from "../../colors";
+import { BaseSvgIcon } from "../BaseSvgIcon";
+
+export const ShoppingBagIcon = () => (
+  <BaseSvgIcon width={20} height={19}>
```

Comment:
Svg icons can be stored in the assests and imported directly in the components

Reply:
resolved in teams